.github/settings.yml

```diff
@@ -38,9 +38,9 @@ branches:
 teams:
   - { name: 'sre',          permission: 'admin' }
   - { name: 'robot-write',  permission: 'admin' }
-  - { name: 'robot',        permission: 'pull' }
-  - { name: 'employees',    permission: 'pull' }
-  - { name: 'security',     permission: 'push' }
+  - { name: 'robot',        permission: 'pull'  }
+  - { name: 'employees',    permission: 'pull'  }
+  - { name: 'eng',          permission: 'push'  }
```

Comment:
Might be worth noting in the description of that team, that it's referenced here. So no one accidentally changes the team name or nukes the team.

Reply:
👍